test/Template/PlatesTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace ZendTest\Expressive\Template;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Zend\Expressive\Template\Plates as PlatesTemplate;
+use League\Plates\Engine;
+
+class PlatesTest extends TestCase
+{
+    public function setUp()
+    {
+        $this->platesEngine = $this->prophesize('League\Plates\Engine');
+    }
+
+    public function testConstructorWithEngine()
+    {
+        $template = new PlatesTemplate($this->platesEngine->reveal());
+        $this->assertTrue($template instanceof PlatesTemplate);
```

Comment:
@samsonasik it's equivalent, I don't want to use assertInstanceOf for namespace reason.